bob-backend/src/core.rs

```diff
@@ -80,7 +80,7 @@ impl Operation {
 
 #[async_trait]
 pub trait BackendStorage: Debug {
```

Comment:
Wouldn't it be easier to make BackendStorage: Debug + Send? We can left default implementations then

Reply:
:facepalm: I tried to add Send, but it didn't work. But adding Sync worked